unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/ChatWidgetHUD/DefaultChatEntry.cs

```diff
@@ -57,6 +58,17 @@ private void Awake()
 
     public override void Populate(ChatEntryModel chatEntryModel)
     {
+        PopulateTask(chatEntryModel);
+    }
+
+    public async UniTask PopulateTask(ChatEntryModel chatEntryModel)
```

Comment:
We should implement this with a CancellationToken

Reply:
I ended up correcting the `PopulateTask()` to be a `UniTaskVoid` instead, since no other method has to wait for it, and since it's usage is a "call and forget" I also added the `Forget()` call.

With all that I think the cancellation token is not needed, since no one will be cancelling that task, same as we do in 2 other places ([here](https://github.com/decentraland/unity-renderer/blob/a222a7a0ac66b438e178419035b505305435aeef/unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/WorldChatWindowHUD/PublicChatChannelController.cs#L168) and [here](https://github.com/decentraland/unity-renderer/blob/a222a7a0ac66b438e178419035b505305435aeef/unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/PrivateChatWindow/PrivateChatWindowController.cs#L160)) using `UniTaskVoid` to await for `UniTask.NextFrame()` inside.

What do you think?

Reply:
In the end I had to keep the `PopulateTask()` as an awaitable `UniTask` to be able to await it in the tests.

Should I add the `CancellationToken` even though no one will ever cancel that "fire and forget" method?

Reply:
yes, we should do the cancellation in the `OnDestroy()` at the very least